example-library/README.md

```diff
@@ -14,15 +14,15 @@
  limitations under the License.
 -->
 # <img src="../logos/logo.svg" width="180">
-
 ## A Tool for Complex and Scalable Data Access Policy Enforcement
 
 # Example Library
```

Comment:
This readme probably wants updating to split up its contents given the split of this module in the codebase

Reply:
Thinking that a lot of this is good stuff for an understanding of what the example does and what to expect you run the test.  That said I think it fit nicely in the  root readme. I have cut and pasted it into that.  See what you think.